src/components/microlearning/styles/VideoDetailPage.scss

```diff
@@ -81,4 +81,32 @@
   .vjs-transcribe-cueline {
     color: #00688d;
   }
+  .logo-cutom-style {
+    max-width: 59px;
+    max-height: 30px;
+    border-radius: 4px;
+    box-shadow: 0 0 8px 0 rgba(0, 0, 0, 0.15), 0px 0px 4px 0px rgba(0, 0, 0, 0.15);
+  }
+}
+.preview-enroll-expand-body {
```

Comment:
Please use bootstrap utility classes instead of custom css.

Reply:
As the styles are custom so we don't have any specific class for this style.

Reply:
Please refer to this document: https://paragon-openedx.netlify.app/foundations/css-utilities

Some Examples are
- height: auto -> h-auto
- overflow: hidden ->. overflow-hidden
- position: relative -> position-relative
- .pl-3 or .pl-4 for padding-left (closest to 1.5rem)